### PR TITLE
Use url_name in urls

### LINKED ISSUE #113: Clean up links
Is there a greater concept behind our urls?
Currently, `A` and `B` is in our link. Do we want to dynamically use `ParticipantConfig.name` in the end?

Blocked by @feeds

### PR DESCRIPTION
**Pull request by Baschdl**
Use the `url_name` from the configuration as `p_type` in the urls etc.

"A" and "B" are still in the dictionaries for programmer's convenience 

Closes #113

### CODE REVIEW

**Comment by Baschdl** [comment]
@feeds Can we use instead of https://github.com/match4everyone/match4everything/blob/2341e57c775cfbb773710d3d17d7f753a5403e08/backend/apps/matching/models/participant.py#L63-L66
something where we first filter for 'p_type' and then filter for 'is_approved'? Alternatively, we would have to also name the groups according to the names in the config.

But we would have similar problems for `if not approver.has_perm(`

**Comment by Baschdl** [comment]
@bjrne `isort` is blocking this one like #175 but possibly one other formatting issue

File: backend/apps/matching/models/participant_info_location.py
Comment (bjrne) [inline backend/apps/matching/models/participant_info_location.py]: This does not look right, correct me if I'm wrong.
```suggestion
    B.url_name: ParticipantInfoLocationB,
```

File: backend/apps/matching/management/commands/createfakeusers.py
Comment (bjrne) [inline backend/apps/matching/management/commands/createfakeusers.py:34]: Lets use
```suggestion
            "--add-{a}".format(a=A.url_name.lower()),
```

**Review by bjrne — DISMISSED** [review]

File: backend/apps/matching/management/commands/createfakeusers.py
Comment (Baschdl) [reply backend/apps/matching/management/commands/createfakeusers.py:34]: It's `url_name = name.lower()` in `A.py` and `B.py`

Comment (bjrne) [reply backend/apps/matching/management/commands/createfakeusers.py:34]: Ah, now i get it.

**Comment by feeds** [comment]
- [x] change the urlconverter
- [x] urlencode for url_name

**Review by feeds — APPROVED** [review]
I added the changes I wanted to see @Baschdl. If you are fine with this please merge :)
And @bjrne you're requests are addressed

**Pull request merged**
